libs/coin-modules/coin-solana/src/prepareTransaction.ts

```diff
@@ -837,4 +841,26 @@ function validateRecipientRequiredMemo(
   }
 }
 
+function isValidStakeCreateAccountCommandDescriptor(
+  commandDescriptor: CommandDescriptor | undefined,
+) {
+  const txCommand = commandDescriptor?.command as StakeCreateAccountCommand | undefined;
+
+  // Ensures commandDescriptor has all required data, avoiding regeneration
+  if (
+    commandDescriptor &&
+    txCommand?.amount &&
+    txCommand.stakeAccRentExemptAmount &&
+    txCommand.fromAccAddress &&
+    txCommand.stakeAccAddress &&
+    txCommand.delegate &&
+    txCommand.seed &&
+    commandDescriptor.fee
+  ) {
+    return true;
+  }
+
+  return false;
```

Comment:
```suggestion
  return commandDescriptor &&
    txCommand?.amount &&
    txCommand.stakeAccRentExemptAmount &&
    txCommand.fromAccAddress &&
    txCommand.stakeAccAddress &&
    txCommand.delegate &&
    txCommand.seed &&
    commandDescriptor.fee;
```

Again, returning boolean literals is really surprising for readers